internal/common/resource.go

```diff
@@ -529,7 +529,7 @@ func defaultStatusFunc(obj client.Object) ResourceStatus {
 			msg := fmt.Sprintf("No pods for deployment %s/%s are running. Expected: %d",
 				objTyped.Namespace,
 				objTyped.Name,
-				objTyped.Status.Replicas)
+				numberOfReplicas)
```

Comment:
nit: Looks like this change belongs into another commit?zz

Reply:
I will move it to another PR.

Reply:
ACK